### ISSUE TITLE
Portrait Main Media Designs

### ISSUE THREAD

**Issue by JamieB-gu** [guardian/dotcom-rendering#4900]
We have some examples of experimental designs to show portrait main media images side-by-side with some of the other article features (headline, standfirst, byline): https://www.theguardian.com/sport/2022/may/16/erin-jackson-interview-speed-skater-winter-olympics

We could bring support for this into the platforms, so that we don't need to achieve this using embeds with client-side JavaScript. This would involve:

1. Building some rough designs to demonstrate how this would work across different `Design`s and `Display`s.
2. Coming up with a way to model this based on the designs. Do we want a new `Display` (e.g. `ImmersivePortrait`)? Do we want to use existing `Display`s coupled with knowledge about the orientation of the main media image?

**Comment by ajbreuer** [comment]
@JamieB-gu This is really interesting, might be worth a quick chat with me, @HarryFischer and @Bryony-Szekeres to see if we can hone some basics. Would certainly be good explore portrait on mobile a bit more

**Comment by HarryFischer** [comment]
Yep I have some ideas on how this can fit into the system - will work on some designs and update

**Comment by paperboyo** [comment]
I think there was some movement on this and we have new version of static/atom js, @HarryFischer? (do you have any examples?)
